app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+const app = express();
+
+const rootRoute = require('./routes/index'); 
+const swaggerSetup = require('./routes/api-docs'); 
+const serverStatusRoute = require('./routes/serverStatus');
+const coursesRoutes = require('./routes/courses'); 
+const errorRoute = require('./routes/error'); 
+
+const errorHandler = require('./utils/errorHandler'); 
+
+const { PORT } = require('./utils/constants'); 
+
+app.use(rootRoute);
+app.use(swaggerSetup);
+app.use(serverStatusRoute);
+app.use('/courses', coursesRoutes);
```

Comment:
You can add all of these routes in the routes folder in the index file, and here you will add only rootRoute and error handler